openwisp_monitoring/check/base/tests/test_models.py

```diff
@@ -0,0 +1,55 @@
+from django.core.exceptions import ValidationError
+from openwisp_monitoring.check.classes import Ping
+from openwisp_monitoring.check.settings import CHECK_CLASSES
+
+
```

Comment:
I just had an idea.. maybe dividing the test classes like we're doing here is not necessary.
Can you test what happens if we keep the tests as they are with the only difference that we load the models using swappable.load_model instead of importing directly?

It should work! If it does we can avoid quite some hassle.